### PR TITLE
super user epic

### PR DESCRIPTION
**Pull request by evan-a-w**

### CODE REVIEW

File: backend/server/src/admin.rs [hunk 1]
```diff
@@ -23,3 +24,14 @@ pub async fn get(user: User, db: Database) -> Json<AdminInfoResponse> {
...
+#[post("/make_superuser", data="<email>")]
+pub async fn make_superuser(_user: SuperUser, db: Database, email: Json<String>) -> Result<(), JsonErr<()>> {
+    db.run(move |conn| {
+        User::get_from_email(conn, &email.into_inner())
+            .map(|user| user.make_superuser(conn).ok())
+            .flatten()
+    })
+    .await
+    .ok_or(JsonErr((), Status::BadRequest))
+}
```
Comment (zaxutic) [inline backend/server/src/admin.rs:37]: does this just let anyone make superusers

Comment (evan-a-w) [reply backend/server/src/admin.rs:37]: Nope, need to already be a superuser. The _user: SuperUser parameter does that check for you epicly

Comment (zaxutic) [reply backend/server/src/admin.rs:37]: ahh i see

**Review by zaxutic — APPROVED** [review]
epic

**Pull request merged**